targets/targets.json

```diff
@@ -6015,6 +6017,7 @@
     },
     "EFM32GG_STK3700": {
         "inherits": ["EFM32GG990F1024"],
+        "macros": ["MBED_TICKLESS"],
```

Comment:
This target inherits from EFM32 which has MBED_TICKLESS

Reply:
Macro removed